controllers/DocumentController.js

```diff
@@ -1,19 +1,19 @@
 const _ = require('lodash')
-
+const Web3 = require('web3')
+const sha3 = new Web3().sha3
```

Comment:
This might be overkill on the hashing. Not sure how worried we are about collisions or having a cryptographically secure hash as the key for documents. Could be slow for large documents